django_fido/views.py

```diff
@@ -30,6 +31,16 @@
 from .models import Authenticator
 from .settings import SETTINGS
 
+try:
+    from fido2.server import AttestationVerifier
+except ImportError:
+    AttestationVerifier = None
+else:
+    from fido2.attestation.base import AttestationResult
+try:
```

Comment:
I'd add some blank lines between try blocks in order to improve readability.

Reply:
Blank lines added.